demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -110,12 +113,54 @@ export function ProductsGrid({ filter, toolbarAction, rowAction }: Props): React
             filterable: false,
             sortable: false,
             renderCell: ({ row }) => {
-                return <GridCellContent primaryText={row.title} secondaryText={row.description} />;
+                const secondaryTextTypeEmptyMessage = <FormattedMessage id="product.overview.secondaryText.type.empty" defaultMessage="No type" />;
```

Comment:
I'd probably wouldn't store this in a variable, since it's only used once

Reply:
Good point 👍🏼 ef848c729bacfa9f91ed39baf8733f05414cbb4b